src/main/webapp/data/userMapData.json

```diff
@@ -1699,6 +1727,7 @@
       "State University of New York Albany",
```

Comment:
Let's remove this because we already have `University at Albany, SUNY`.

Reply:
Done.